xmppserver/src/main/java/org/jivesoftware/util/cache/CacheMXBean.java

```diff
@@ -0,0 +1,4 @@
+package org.jivesoftware.util.cache;
+
+public interface CacheMXBean {
```

Comment:
Probably a bit of JavaDoc here explaining what this is for may help.

Reply:
That should not be in this commit at all, I think. I think I messed up.

Reply:
Should be gone now.